ceph-extra-exporter/README.md

```diff
@@ -3,6 +3,33 @@ ceph-extra-exporter
 
 `ceph-extra-exporter` exposes extra metrics using some Ceph commands.
 
+## How to work
+
+ceph-extra-exporter get metrics using `ceph`, `rados-gw-admin`, and related commands and export metric values like as follows.
+
+- Run `ceph` commands etc written in the code.
+- Modify output JSON to specify format using jq filter.
```

Comment:
I could not understand the meaning of this explanation.
I would be happy if you could tell me the specific lines of Go code that this explanation refers to.

Reply:
@llamerada-jp Could you give me a reply to this comment?

Reply:
@llamerada-jp Sorry, but I just wanted to see the code to understand this explanation. I think you need not to write the Go source info in the document itself.

Anyway, now I believe I understand the meaning. How about changing this line as follows?

```suggestion
- Modify the output JSON to the specified format using jq filter.
```